### ISSUE TITLE
Screen frame changes can able to see with naked eyes

### ISSUE THREAD

**Issue by Pandurangbhor** [TomSchimansky/CustomTkinter#979]
I have made a application with custumtkinter python , when it changes a screen we can see how it is changes , like each frame comes and changes its position until final position takes, within 1 sec

**Comment by JardarIversen** [comment]
Yes, I have the same problem. Also in a menu where is disable buttons, the button text for the other buttons become grey for about a second, even though i do nothing to them
